### ISSUE TITLE
RTD docs build fails

### ISSUE THREAD

**Issue by orbeckst** [Becksteinlab/MDPOW#153]
RTD [build 3671699](https://readthedocs.org/projects/mdpow/builds/13671699/) fails, apparently issues with compiling against the numpy installation

```
Searching for mdanalysis
Reading https://pypi.org/simple/mdanalysis/
Downloading https://files.pythonhosted.org/packages/43/64/5fc4d327f242130f983b9a1ded4a406191385370dbc55a7be99f8ead126d/MDAnalysis-1.1.1.tar.gz#sha256=edfd5a8ac6cbaa8d02f505ac3e68d5a1866644a2d392a8f20e279c43e0886253
Best match: MDAnalysis 1.1.1
Processing MDAnalysis-1.1.1.tar.gz
Writing /tmp/easy_install-rqktB8/MDAnalysis-1.1.1/setup.cfg
Running MDAnalysis-1.1.1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-rqktB8/MDAnalysis-1.1.1/egg-dist-tmp-KC9XPd
warning: no previously-included files matching 'doc/html/html' found anywhere in distribution
warning: no previously-included files matching '*.pyc' found anywhere in distribution
warning: no previously-included files matching '*.pyo' found anywhere in distribution
warning: no previously-included files matching '*.pyd' found anywhere in distribution
warning: no previously-included files matching '*~' found anywhere in distribution
warning: no previously-included files matching '.git' found anywhere in distribution
Attempting to autodetect OpenMP support... Compiler supports OpenMP
Will not attempt to use Cython.
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/mdpow/checkouts/latest/setup.py", line 60, in <module>
    zip_safe=True,
...
  File "/home/docs/.pyenv/versions/2.7.18/lib/python2.7/distutils/command/build_ext.py", line 496, in build_extension
    include_dirs=ext.include_dirs,
  File "/tmp/easy_install-rqktB8/MDAnalysis-1.1.1/setup.py", line 170, in include_dirs
    
  File "/tmp/easy_install-rqktB8/MDAnalysis-1.1.1/setup.py", line 196, in get_numpy_include
    
  File "/home/docs/.pyenv/versions/2.7.18/lib/python2.7/site-packages/numpy/__init__.py", line 142, in <module>
    from . import core
  File "/home/docs/.pyenv/versions/2.7.18/lib/python2.7/site-packages/numpy/core/__init__.py", line 40, in <module>
    from . import multiarray
  File "/home/docs/.pyenv/versions/2.7.18/lib/python2.7/site-packages/numpy/core/multiarray.py", line 13, in <module>
    from . import overrides
  File "/home/docs/.pyenv/versions/2.7.18/lib/python2.7/site-packages/numpy/core/overrides.py", line 46, in <module>
    """)
RuntimeError: implement_array_function method already has a docstring
```

**Comment by orbeckst** [comment]
Possible source: https://github.com/numpy/numpy/issues/14384

**Comment by orbeckst** [comment]
Possible workaround: downgrade to Python 1.15.4 or upgrade to 1.19.x (if this still supports Python 2.7)

https://stackoverflow.com/questions/65159846/runtimeerror-implement-array-function-method-already-has-a-docstring-trying

**Comment by orbeckst** [comment]
Fixed in PR #167 . 

see https://mdpow.readthedocs.io/en/latest/
